tests/roles/run_tests/vars/main.yml

```diff
@@ -162,7 +162,7 @@ pivot_actions:
     - "pivoting"
     - "upgrading"
 repivot_actions:
-    - "ci_test_provision"
```

Comment:
trying to understand removal of this line, can you elaborate why this action is removed?

Reply:
and why not changing the previous ci_test_provision entries in other actions?

Reply:
this seems incorrect to me.

Reply:
Thanks for the explanation offline, it should work as I understand now, but my assumption was we are going to do it ansible way i.e. adding tasks at the beginning of move_back.yml. Anyways, if this works I am fine with this approach as well, as long as it doesnt break local as well as CI tests.